### PR TITLE
draft: 2024-03-07-february-2024-roundup.md

### PR DESCRIPTION
**Pull request by geekigirlx**
Can you please double check all the descriptions and make sure they are accurate if they don't have a name next to them? (Minus François, because I wrote his description)

Also added some videos/blog posts (no name attached) - we can remove any that aren't worthy. 

Note: not many projects and tools this time around. 

**Should we announce Memfault Launch Week in the news and announcements? I am leaning towards yes, and point them to launch week landing page. ... actually yes let me create PR and then edit.

### CODE REVIEW

**Review by geekigirlx — COMMENTED** [review]
launch week

**Review by gminn — APPROVED** [review]
Looks good, just a date fix. Weirdly the cloudfare preview is not showing up 🤔

File: _posts/2024-03-06-february-2024-roundup.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+- [**Board Porting Guide — Zephyr Project Documentation**](https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#transition-to-the-current-hardware-model)<br>
+There’s a new Zephyr change for how users define their hardware that will be in v3.7 LTS, and this migration guide helps move to these new features. - Eric
+
+- [**Get up to speed with partial clone and shallow clone - The GitHub Blog**](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/)<br>
+A good explainer on partial and shallow clones, how they work, and when to use different forms. Great to help speed up downloading large vendor HALs. - Eric
+
+- [**What is Name Mangling in C++? – Abstract Expression**](https://abstractexpr.com/2023/01/03/what-is-name-mangling-in-cpp/)<br>
+Great explanation of name mangling (encoding a name and a signature to a symbol name) and how to demangle.
+
+- [**Making the most of your Shells History on Linux and macOS – Abstract Expression**](https://abstractexpr.com/2024/02/18/making-the-most-of-your-shells-history-on-linux-and-macos/)<br>
```
Comment (gminn) [inline _posts/2024-03-06-february-2024-roundup.md:66]: Since this isn't directly tied to embedded, I think we should remove it

Comment (geekigirlx) [reply _posts/2024-03-06-february-2024-roundup.md:66]: @gminn the Shells History or Name Mangling one? Thank you so much for double checking all these!!

Comment (gminn) [reply _posts/2024-03-06-february-2024-roundup.md:66]: Shells History

**Pull request merged**